src/dotnetcore31/AzureFunctions.Durable.Chaining/GitHub/GitHubInfoOrchestrator.cs

```diff
@@ -0,0 +1,69 @@
+using System.Linq;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.DurableTask;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.Extensions.Logging;
+using Octokit;
+
+namespace Durable.Chaining.GitHub
+{
+    public static class GitHubInfoOrchestrator
+    {
+        [FunctionName("GitHubInfoOrchestrator")]
+        public static async Task<string> RunOrchestrator(
+            [OrchestrationTrigger] IDurableOrchestrationContext context)
+        {
+            var input = context.GetInput<string>();
+
+            var userName = await context.CallActivityAsync<string>("GitHubInfoOrchestrator_GetRepositoryDetailsByName", input);
+
+            var userDetails = await context.CallActivityAsync<string>("GitHubInfoOrchestrator_GetUserDetailsByName", userName);
+
+            //context.SetOutput(userDetails);
```

Comment:
I think this commented out code can be removed. (I actually didn't know there was a SetOutput method! )